generators/Output.fs

```diff
@@ -0,0 +1,224 @@
+module Generators.Output
+
+open System
+open System.Collections.Generic
+open System.IO
+open System.Reflection
+open FSharp.Reflection
+open DotLiquid
+open DotLiquid.FileSystems
+open Newtonsoft.Json.Linq
+open Input
+
+let indent level value = 
+    let indentation = String.replicate level "    "
+    sprintf "%s%s" indentation value
+
+let parenthesize value = sprintf "(%s)" value
+
+let backwardPipe value = sprintf "<| %s" value
+
+let backwardPipeConditional test value = if test value then backwardPipe value else value
+
+let addTypeAnnotation ty value = sprintf "%s: %s" value ty
```

Comment:
I'm still wondering what `ty` is :)

Reply:
It's the type (e.g. `"int list'`). I'll rename.